resolwe/flow/tests/test_filtering.py

```diff
@@ -551,10 +552,10 @@ def _apply_filter(self, filters, expected):
         self.assertCountEqual(filtered.qs, expected)
 
     def test_descriptor_completed(self):
-        self._apply_filter({'descriptor_completed': 'true'}, [self.entity_1])
-        self._apply_filter({'descriptor_completed': '1'}, [self.entity_1])
-        self._apply_filter({'descriptor_completed': 'false'}, [self.entity_2])
-        self._apply_filter({'descriptor_completed': '0'}, [self.entity_2])
+        self._apply_filter({'descriptor_completed': True}, [self.entity_1])
```

Comment:
Filters are normally passed as a string from a client, so tests have to work the sam way.